src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionToolOptions.cs

```diff
@@ -4,368 +4,62 @@
-
-        public RoslynInsertionToolOptions WithBuildConfig(string buildConfig) => Update(buildConfig: buildConfig);
-
-        public RoslynInsertionToolOptions WithInsertionBranchName(string insertionBranchName) => Update(insertionBranchName: insertionBranchName);
-
-        public RoslynInsertionToolOptions WithBuildDropPath(string buildDropPath) => Update(buildDropPath: buildDropPath);
-
-        public RoslynInsertionToolOptions WithSpecificBuild(string specificBuild) => Update(specificBuild: specificBuild);
-
-        public RoslynInsertionToolOptions WithInsertCoreXTPackages(bool insertCoreXTPackages) => Update(insertCoreXTPackages: insertCoreXTPackages);
-
-        public RoslynInsertionToolOptions WithInsertDevDivSourceFiles(bool insertDevDivSourceFiles) => Update(insertDevDivSourceFiles: insertDevDivSourceFiles);
-
-        public RoslynInsertionToolOptions WithInsertWillowPackages(bool insertWillowPackages) => Update(insertWillowPackages: insertWillowPackages);
-
-        public RoslynInsertionToolOptions WithInsertionName(string insertionName) => Update(insertionName: insertionName);
-
-        public RoslynInsertionToolOptions WithUpdateCoreXTLLibraries(bool updateCoreXTLibraries) => Update(updateCoreXTLibraries: updateCoreXTLibraries);
-
-        public RoslynInsertionToolOptions WithUpdateAssemblyVersions(bool updateAssemblyVersions) => Update(updateAssemblyVersions: updateAssemblyVersions);
-
-        public RoslynInsertionToolOptions WithCreateDummyPr(bool createDummyPr) => Update(createDummyPr: createDummyPr);
 
-        public RoslynInsertionToolOptions WithUpdateExistingPr(int updateExistingPr) => Update(updateExistingPr: updateExistingPr);
-
-        public RoslynInsertionToolOptions WithOverwritePr(bool overwritePr) => Update(overwritePr: overwritePr);
-
-        public RoslynInsertionToolOptions WithLogFileLocation(string logFileLocation) => Update(logFileLocation: logFileLocation);
-
-        public RoslynInsertionToolOptions WithClientId(string clientId) => Update(clientId: clientId);
-
-        public RoslynInsertionToolOptions WithClientSecret(string clientSecret) => Update(clientSecret: clientSecret);
-
-        public RoslynInsertionToolOptions WithTitlePrefix(string titlePrefix) => Update(titlePrefix: titlePrefix);
-
-        public RoslynInsertionToolOptions WithCreateDraftPr(bool createDraftPr) => Update(createDraftPr: createDraftPr);
-
-        public RoslynInsertionToolOptions WithSetAutoComplete(bool setAutoComplete) => Update(setAutoComplete: setAutoComplete);
-
-        public RoslynInsertionToolOptions WithCherryPick(ImmutableArray<string> cherryPick) => Update(cherryPick: cherryPick);
-
-        public RoslynInsertionToolOptions WithSkipCoreXTPackages(string skipCoreXTPackages) =>
-            Update(skipCoreXTPackages: (skipCoreXTPackages ?? string.Empty).Split(',').Select(packageName => packageName.Trim()).Where(packageName => !string.IsNullOrEmpty(packageName)).ToImmutableArray());
```

Comment:
This logic was moved to `ParseSkipCoreXTPackages` in `RoslynInsertionToolCommandline.cs`.